dspace-api/src/main/java/org/dspace/statistics/util/NoDotOldFilter.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * 
+ */
+package org.dspace.statistics.util;
+
+import java.io.File;
+import java.io.FileFilter;
+
+/**
+ * @author Ulrich Hahn
+ *
+ */
+public class NoDotOldFilter implements FileFilter {
+
+	/**
+	 * All files with ending .old are fitered out.
+	 * Motivation: each "ant update" run backs up changed config files with .old extensions.
+	 * These files were read unexpectedly in addition to the current config files.
+	 */
+
+	/* (non-Javadoc)
+	 * @see java.io.FileFilter#accept(java.io.File)
+	 */
```

Comment:
I would merge the @see into the doc comment above and get rid of this non-doc comment, so the javadocs are built properly.